py/server/deephaven/table.py

```diff
@@ -62,6 +65,86 @@ def _j_py_script_session() -> _JPythonScriptSession:
         return None
 
 
+def _encode_signature(fn: Callable) -> str:
```

Comment:
I made a prior comment about creating a separate signature class in java.  I see that you are using the string to encode in python and then hand to java.  Consider the class in Java to wrap the string and clean up the use, while still using the string to communicate between the languages.